util/device_sw_utils/extract_sw_logs.py

```diff
@@ -157,7 +157,9 @@ def get_str_at_addr(str_addr, addr_strings):
     for addr in addr_strings.keys():
         if addr <= str_addr < addr + len(addr_strings[addr]):
             return addr_strings[addr][str_addr - addr:].strip()
-    raise KeyError("string at addr {} not found".format(str_addr))
+    raise KeyError(f"string at addr {str_addr} not found"
+                   "Are you trying to print (unsupported) whitespace? "
```

Comment:
I feel like this is probably only _one_ of many other possible diagnoses. Not sure it is useful to call this out specifically. Besides, `LOG_INFO("\n")` should absolutely work! I will be worth investigating this further.